geode-redis/src/integrationTest/java/org/apache/geode/redis/StringsIntegrationTest.java

```diff
@@ -270,10 +586,10 @@ public void testGetSet_whenWrongType_shouldReturnError() {
 
   @Test
   public void testSet_keyExistsWithDifferentDataType_returnsRedisDataTypeMismatchException() {
```

Comment:
i think this test belongs in the HSET tests, not in the strings.  
Consider moving it?
Another option might be moving all the tests that re-write keys into a single test, so that context is in one place.

Reply:
also, we might want to test that set overrides the hset, if we are not doing so already

Reply:
done